bref/symfony-bridge/0.1/manifest.json

```
@@ -0,0 +1,8 @@
+{
+    "copy-from-recipe": {
+        "serverless.yaml": "serverless.yaml"
```

Should be `serverless.yml`

---

Both are valid for Serverless and the choice to use `.yaml` may be because of the [Validation](https://github.com/symfony/recipes#validation) applied to PRs here 

> YAML files suffix must be `.yaml`, not `.yml`

But the standard for Serverles Framework is indeed `.yml`. Maybe this could be used as a "workaround"

```suggestion
        "serverless.yml": "serverless.yaml"
```

---

BTW it's already accepted for `docker-compose.yml` if that may help and I think we are in the same situation because it's not Symfony config we are talking about but standard in a third-party tool

See for example : https://github.com/symfony/recipes/blob/master/doctrine/doctrine-bundle/1.6/manifest.json#L24

---

Yes, i used the `.yaml` extension to pass validation!

@t-richard wouldn't it be rather
```suggestion
        "serverless.yaml": "serverless.yml"
```
?

The other solution would be to store the `serverless.yml` into the package and use `copy-from-package` instead.

---

Yes it would be `"serverless.yaml": "serverless.yml"` :facepalm: 

IMHO it makes more sense to have all here